src/notification.h

```diff
@@ -86,8 +91,17 @@ struct notification {
 
         enum markup_mode markup;
         char *format;
+
+        enum mouse_action_type script_choice;
```

Comment:
rather than having this choice variable inside notification I would rather add mouse_action parameter to run_script function. this seems a bit more clean and you don't risk forgetting to set the type of action beforehand